sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -292,6 +292,22 @@ void WinHttpTransport::Upload(std::unique_ptr<_detail::HandleManager>& handleMan
   }
 }
 
+static inline std::string GetHeadersAsString(Azure::Core::Http::Request const& request)
+{
+  std::string requestHeaderString;
+
+  for (auto const& header : request.GetHeaders())
```

Comment:
consider using ostringstream, it should be more performance-effective for building strings

Reply:
Good idea, will do that separately.